.github/workflows/images.yml

```diff
@@ -1,9 +1,7 @@
 on:
   workflow_call:
     inputs:
-      test:
-        required: false
-        type: boolean
+      test: { required: false, type: boolean }
```

Comment:
👎

Reply:
actually, is this `test` input even used in this workflow @0x2b3bfa0 ?

Reply:
It's not 🙈

Reply:
```suggestion
```